po/LINGUAS

```diff
@@ -32,4 +33,3 @@ kab
 si
 nl
 fi
-pt
```

Comment:
Then, could you sort all entries in this file?

Reply:
@yuwata Thank you.
Yes, of course. I have already ordered the lang codes.